### PR TITLE
Completed donation pg's impact section and team contact strip

### PR DESCRIPTION
**Pull request by emilyschang**

### CODE REVIEW

File: css/donate.css [hunk 1]
```diff
@@ -75,57 +81,53 @@ h2 {
...
-
-.donate-option2 {
-  background: #ffffff;
-  border: 6px solid #a88aa8;
-  padding: 3%;
-  padding-top: 5%;
-  text-align: center;
-  width: 100%;
-  height: 100%; }
+  color: #82ecec; }
```
Comment (julesparra) [inline css/donate.css:99]: can you update this blue to match that of the first two social buttons & intercom bot? #6fcbdc

File: css/donate.css [hunk 2]
```diff
@@ -75,57 +81,53 @@ h2 {
...
+
+.donate-option1, .donate-option2, .donate-option3 {
+  background: white;
+  border-radius: 33px;
+  padding: 11%;
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 80%;
+  height: 100%;
+  min-height: 315px; }
```
Comment (julesparra) [inline css/donate.css:115]: let's decrease slightly to 280px (to the point where height of all boxes match while being as short as possible)

File: donate.html [hunk 3]
```diff
@@ -202,15 +202,14 @@ <h1 class="our-donor">Our Donor Promise</h1>
...
+      <div>
+        <h1 class="impact-header">Your Donation's Impact</h1>
+      </div>
 
       <!-- body -->
-      <div class="row" style="padding-bottom: 3%;">
+      <div class="row align-items-center justify-content-center">
         <div class="col-sm">
           <div class="donate-option1">
               <h2 class="donate-option1-header">$20</h2>
```
Comment (julesparra) [inline donate.html:215]: gen asked us to change this highest option to $50

File: scss/donate.scss [hunk 4]
```diff
@@ -7,16 +7,23 @@ $light-lavender: #cca8c1;
...
   align-items: center;
-  padding: 7px 35px;
+  color: white;
+  background-color: $lavender;
+  border: none;
+  border-radius: 50px;
   font-size: 35px;
   font-weight: bold;
-  border-radius: 6px;
+  padding: 7px 35px;
```
Comment (julesparra) [inline scss/donate.scss:20]: button looks a bit too big

File: scss/donate.scss [hunk 5]
```diff
@@ -7,16 +7,23 @@ $light-lavender: #cca8c1;
...
-  color: #fff;
-  border: none;
-  background-color: #ff5789;
   align-items: center;
-  padding: 7px 35px;
+  color: white;
+  background-color: $lavender;
+  border: none;
+  border-radius: 50px;
   font-size: 35px;
```
Comment (julesparra) [inline scss/donate.scss:18]: i think slightly smaller text looks better - 30px

**Review by julesparra — CHANGES_REQUESTED** [review]
nice! only small things to change for now - may be updating impact tiers, will lyk

File: css/donate.css (hunk 2 above)
Comment (emilyschang) [reply css/donate.css:115]: I think the min height for all to match is 315px if I leave padding as 11%. If you want the boxes to be more square-shaped (280px min height), I'm going to have to change padding to 7%. I think 8% padding and 290px min-height also works out pretty well.

**Review by julesparra — APPROVED** [review]
lookin good ⭐

**Pull request merged**